### PR TITLE
test:add RejectedProxyInvocationHandlerTest

### LINKED ISSUE #1175: [TASK] RejectedProxyInvocationHandler add unit test.
## 需求建议

为 RejectedProxyInvocationHandler 编写单元测试

### PR DESCRIPTION
**Pull request by z-y-hang**
Fixes #1175

Changes proposed in this pull request:
- 

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: hippo4j-core/pom.xml [hunk 1]
```diff
@@ -20,5 +20,9 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
```
Comment (magestacks) [inline hippo4j-core/pom.xml:25]: Add scope as test

File: hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.context.ApplicationContext;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.concurrent.atomic.AtomicLong;
+
+
```
Comment (magestacks) [inline hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java]: Remove extra blank lines.

File: hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    @Test
+    public void testInvoke() throws Throwable {
+        Object[] mockArgs = new Object[] {"arg1", "arg2"};
+        MockedStatic<ApplicationContextHolder> mockedStatic = Mockito.mockStatic(ApplicationContextHolder.class);
+        mockedStatic.when(ApplicationContextHolder::getInstance).thenReturn(applicationContext);
+        mockedStatic.when(() -> ApplicationContextHolder.getBean(ThreadPoolCheckAlarm.class)).thenReturn(mockAlarmHandler);
+        Mockito.doNothing().when(mockAlarmHandler).asyncSendRejectedAlarm("test-pool");
+        handler.invoke(null, mockMethod, mockArgs);
+
```
Comment (magestacks) [inline hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java]: Remove extra blank lines.

File: hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        Mockito.doThrow(new InvocationTargetException(new Throwable())).when(mockMethod).invoke(target, mockArgs);
+        Assertions.assertThrows(Throwable.class, () -> handler.invoke(null, mockMethod, mockArgs));
```
Comment (iwangjie) [inline hippo4j-core/src/test/java/cn/hippo4j/core/proxy/RejectedProxyInvocationHandlerTest.java:52]: add rejectCount after value check.

**Review by iwangjie — CHANGES_REQUESTED** [review]

**Review by magestacks — APPROVED** [review]

**Review by magestacks — APPROVED** [review]

**Review by iwangjie — APPROVED** [review]

**Pull request merged**